docs/appcontext.rst

```diff
@@ -5,31 +5,37 @@ The Application Context
 
 .. versionadded:: 0.9
 
-One of the design ideas behind Flask is that there are two different
-“states” in which code is executed.  The application setup state in which
-the application implicitly is on the module level.  It starts when the
-:class:`Flask` object is instantiated, and it implicitly ends when the
-first request comes in.  While the application is in this state a few
-assumptions are true:
+One of the design ideas behind Flask is that there are at least two 
+different “states” in which code is executed:
+
+1. The application setup state, in which the application implicitly is 
+on the module level.  
+
+This state starts when the :class:`Flask` object is instantiated, and 
```

Comment:
I think I just fixed this [here](https://github.com/pallets/flask/pull/1887/commits/863e5cca1b498fb3170981892e820df97f4200e7) .